SPECS/cert-manager/cert-manager.spec

```diff
@@ -1,96 +1,117 @@
 Summary:        Automatically provision and manage TLS certificates in Kubernetes
 Name:           cert-manager
-Version:        1.5.3
-Release:        2%{?dist}
+Version:        1.7.3
+Release:        1%{?dist}
 License:        ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/jetstack/cert-manager
 #Source0:       https://github.com/jetstack/%{name}/archive/refs/tags/v%{version}.tar.gz
 Source0:        %{name}-%{version}.tar.gz
 # Below is a manually created tarball, no download link.
-# We're using pre-populated external dependencies from this tarball, since network is disabled during build time.
-#   1. wget https://github.com/jetstack/%{name}/archive/refs/tags/v%{version}.tar.gz -o %%{name}-%%{version}.tar.gz
-#   2. tar -xf %%{name}-%%{version}.tar.gz
-#   3. cd %%{name}-%%{version}
-#   4. patch -p1 < Fix-dependency-checksum.patch
-#   5. mkdir -p BAZEL_CACHE
-#   6. bazel fetch --repository_cache=BAZEL_CACHE //...
-#   7. tar  --sort=name \
-#           --mtime="2021-04-26 00:00Z" \
-#           --owner=0 --group=0 --numeric-owner \
-#           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
-#           -cf %%{name}-%%{version}-vendor.tar.gz BAZEL_CACHE
-Source1:        %{name}-%{version}-vendor.tar.gz
-# Below is a manually created tarball, no download link.
 # We're using pre-populated GO dependencies from this tarball, since network is disabled during build time.
 #   1. wget https://github.com/jetstack/%{name}/archive/refs/tags/v%{version}.tar.gz -o %%{name}-%%{version}.tar.gz
 #   2. tar -xf %%{name}-%%{version}.tar.gz
 #   3. cd %%{name}-%%{version}
-#   4. go mod download
-#   5. cd $HOME
+#   4. go mod vendor
 #   5. tar  --sort=name \
-#           --mtime="2021-04-26 00:00Z" \
+#           --mtime="2022-08-10 00:00Z" \
```

Comment:
Let's not change the `mtime`. This is just some constant date we use everywhere else for the sake of getting a reproducible tarball. Keeping the date always in line with the date the tarball was introduced is probably not worth the hustle and the potential issues we'd hit, if at some point we'd like to get a tarball for an earlier version of the package (who knows what date one might have used at that point?).

Reply:
Fair.